actions/class.ItemContent.php

```diff
@@ -34,6 +34,7 @@
 use oat\taoItems\model\media\AssetTreeBuilder;
 use oat\taoItems\model\media\AssetTreeBuilderInterface;
 use oat\taoItems\model\media\ItemMediaResolver;
+use oat\taoMediaManager\model\MediaSource;
```

Comment:
You cannot add this here, cause it is part of taoMediaManager and this extension is not coupled to it.